deps/rabbit/src/rabbit_mnesia_rename.erl

```diff
@@ -159,6 +165,9 @@ finish(FromNode, ToNode, AllNodes) ->
               [FromNode, ToNode, node()])
     end.
 
+nodes_running(Nodes) ->
+    [N || N <- Nodes, rabbit:is_running(N)].
```

Comment:
The same function exists in `rabbit_nodes` as `rabbit_nodes:all_running/0`

Reply:
If I understand correctle, they don't have the same definition of "running". `rabbit_nodes:all_running/0` returns a list of nodes where Mnesia is running, whereas `rabbit_mnesia_rename:nodes_running/1` filters nodes where RabbitMQ is running.

Reply:
Fair enough, we should port this to `rabbit_nodes` under a slightly different name, e.g. `rabbit_nodes:all_with_rabbitmq_running/0`.

Reply:
And perhaps alias `rabbit_nodes:all_running/0` to `rabbit_nodes:all_with_mnesia_running/0`. Eventually it can become  `rabbit_nodes:all_with_khepri_running/0` if we need one.

Reply:
Good idea. I did what you suggest but used a slightly shorted name, `all_running_{mnesia,rabbitmq}()`.